rust/openvasd/src/config.rs

```diff
@@ -422,8 +422,9 @@ impl Config {
                 clap::Arg::new("enable-get-scans")
                     .env("ENABLE_GET_SCANS")
                     .long("enable-get-scans")
-                    .action(ArgAction::SetTrue)
-                    .help("enable get scans endpoint"),
+                    .num_args(0..=1)
+                    .default_missing_value("true")
```

Comment:
This is just a nitpick and you can ignore it: I think if you add ` .value_parser(clap::builder::BoolValueParser::new())` you can set `default_missing_value(true)` and spare the `matches!(...)` check later on.

Reply:
Hey. That did the trick. Thanks!